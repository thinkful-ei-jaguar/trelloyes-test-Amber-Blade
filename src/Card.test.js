import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card.js';
import renderer from 'react-test-renderer';

let title = 'Test title';
let content = 'String of content';

it('this card works', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card title={title} content={content} />, div);
    ReactDOM.unmountComponentAtNode(div);
});


it('renders the UI as expected', () => {
    const tree = renderer.create(<Card title={title} content={content} />)
    .toJSON();
    expect(tree).toMatchSnapshot();  
    });


