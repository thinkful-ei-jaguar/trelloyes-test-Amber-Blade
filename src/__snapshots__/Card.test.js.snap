// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Test title
  </h3>
  <p>
    String of content
  </p>
</div>
`;
